doc/getting-started/getting-started.ipynb

```diff
@@ -2,9 +2,44 @@
  "cells": [
   {
```

Comment:
Could you remove this file from PR? It blocks merging.

Reply:
Yes, I will remove right now.